homeassistant/components/zwave_js/__init__.py

```diff
@@ -163,6 +165,7 @@ def async_on_value_notification(notification: ValueNotification) -> None:
     def async_on_notification(notification: Notification) -> None:
         """Relay stateless notification events from Z-Wave nodes to hass."""
         device = dev_reg.async_get_device({get_device_id(client, notification.node)})
+        assert device
```

Comment:
```suggestion
        if not device:
            return
```

Reply:
Modified to explicitly silence the MyPy errors for now, let's do proper error handling + tests in a separate PR